dopamine/discrete_domains/run_experiment.py

```diff
@@ -602,6 +602,9 @@ def _run_iterations(self):
     """
     # TODO(aarg): Change the thread management to an implementation with queues.
     threads = []
+    # TODO(westurner): See how to refactor the code to avoid setting an internal
```

Comment:
Just by the by, the style guide wants a blank line before the `_init__`.

> C6020:567: Need 1 blank line before def [g-wrong-blank-lines]
> go/gpylint-faq#g-wrong-blank-lines

Reply:
Done